rdrf/rdrf/forms/fields/calculated_functions.py

```diff
@@ -513,6 +513,30 @@ def FHDeathAge_inputs():
 
 ################ END OF FHDeathAge ################################
 
+################ BEGINNING OF RLDRDeathAge ################################
+
+def RLDRDeathAge(patient, context):
```

Comment:
This code is dead , in fact - so remove ads FHDeathAge is the cde being used ...